venues/thecvf.com/ECCV/2020/Conference/python/secondary_ACs.py

```diff
@@ -0,0 +1,324 @@
+                'label': {'value-regex': '.*'}
+            }
+        }))
+    print('Posted aggregate score invitation:', aggregate_score_invitation.id)
+
+    map_submissions = {note.number: note for note in openreview.tools.iterget_notes(client, invitation = 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission')}
+    
+    all_meta_reviews = list(openreview.tools.iterget_notes(client, invitation='thecvf.com/ECCV/2020/Conference/Paper[0-9]*/-/Meta_Review$'))
+    print('Found {} official meta reviews\n'.format(len(all_meta_reviews)))
+    set_reject_papers = set()
+    for meta in all_meta_reviews:
+        paper_num = int(meta.invitation.split('Paper')[1].split('/')[0])
+        if paper_num in map_submissions:
+            if meta.content['recommendation'] == 'Reject':
+                set_reject_papers.add(paper_num)
+
+    custom_demand_invitation = client.post_invitation(openreview.Invitation(
+        id='thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs/-/Custom_User_Demands',
+        signatures=['thecvf.com/ECCV/2020/Conference'],
+        readers=[
+            'thecvf.com/ECCV/2020/Conference',
+            'thecvf.com/ECCV/2020/Conference/Area_Chairs'],
+        writers=['thecvf.com/ECCV/2020/Conference'],
+        invitees=['thecvf.com/ECCV/2020/Conference'],
+        reply = {
+            'readers': {'values-copied': [
+                'thecvf.com/ECCV/2020/Conference',
+                'thecvf.com/ECCV/2020/Conference/Area_Chairs',
+                '{tail}']},
+            'nonreaders': {'values-regex': 'thecvf.com/ECCV/2020/Conference/Paper.*/Authors'},
+            'writers': {'values': ['thecvf.com/ECCV/2020/Conference']},
+            'signatures': {'values': ['thecvf.com/ECCV/2020/Conference']},
+            'content': {
+                'head': {
+                    'query' : {'invitation': 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission'},
+                    'type': 'Note'
+                },
+                'tail': {
+                    'query' : {'id': 'thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs'},
+                    'type': 'Profile'
+                },
+                'weight': {
+                    'value-regex': '[-+]?[0-9]*\\.?[0-9]*'
+                },
+                'label': {
+                    'value-regex': '.*'
+                }
+            }
+        }
+    ))
+    print('Posted invitation {}\n'.format(custom_demand_invitation.id))
+
+    # print('\nDeleting old edges for invitation:"thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs/-/Custom_User_Demands"\n')
+    # client.delete_edges(invitation='thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs/-/Custom_User_Demands')
+    # time.sleep(10)
+
+    secondary_ac_demands = []
+    total_review_demand = 0
+    print('Accumulating Custom_User_Demand edges')
+    for paper_num, sub in tqdm(map_submissions.items()):
+        if paper_num not in set_reject_papers:
```

Comment:
only papers that are no clear reject should get a secondary area chair.

Reply:
There are three cases wrt clear reject cases: Papers with no meta reviews, Papers with meta review with `clear_reject = yes`, Papers with meta review with `clear_reject = no`.

It just seemed straightforward to collect the `clear_reject = yes` cases and not create edge for these rather than first collecting the `clear_reject = no` cases and then adding to this collection the papers with no meta-reviews.

Let me know if you prefer creating a collection of non-rejected papers and I can do that.